File: app/src/main/java/ru/buylist/data/Category.java
package ru.buylist.data;

import java.util.UUID;

public class Category {
    private long id;
    private String name;
    private String color;

    public Category() {
        id = System.currentTimeMillis();
    }

    public Category(long id) {
        this.id = id;
    }

    public long getId() {
        return id;
    }

    public String getName() {
        return name;
    }

    public void setName(String name) {
        this.name = name;
    }

    public String getColor() {
        return color;
    }

    public void setColor(String color) {
        this.color = color;
    }

    @Override
    public String toString() {
        return this.name;
    }
}

File: app/src/main/java/ru/buylist/data/entity/Category.kt
package ru.buylist.data.entity

import androidx.room.Entity
import androidx.room.PrimaryKey

@Entity(tableName = "categories")
data class Category(
        @PrimaryKey var id: Long = System.currentTimeMillis(),
        var name: String = "Без категории",
        var color: String = "#8A000000"
) {
    override fun toString(): String = name
}
